packages/tools/client-debugger/client-debugger-view/src/components/RenderLabel.tsx

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { Primitive } from "@fluid-tools/client-debugger";
+// eslint-disable-next-line import/no-internal-modules
+import { TreeItemLayout } from "@fluentui/react-components/unstable";
+import { tokens } from "@fluentui/react-components";
+
+/**
+ * Input props to {@link RenderLabel}
+ */
+export interface RenderLabelProps {
+	/**
+	 * Key of the child node from Record {@link VisauTree}.
+	 */
+	label: string;
+
+	/**
+	 * Type of the object.
+	 */
+	nodeTypeMetadata?: string | undefined;
+
+	/**
+	 * Nodekinds to filter rendering pattern in {@link TreeDataView}.
+	 */
+	nodeKind?: string;
+
+	/**
+	 * Size of the children inside the data.
+	 */
+	itemSize?: Primitive;
```

Comment:
Nit: Rather than taking in `itemSize` specifically, what about taking in the entire `metadata` json property and displaying stats more generally? Don't worry about doing this in this PR, but it would be nice to be able to display our other metadata stats in a general way. Probably worth a separate task.

Reply:
I will create a separate task for passing `metaData` for each node so that we can render more data in our container data panel.